lm_tournament_eval/__main__.py

```diff
@@ -133,10 +142,40 @@ def run_tournament():
 
     logging.info(f"Selected Tasks: {task_names}")
 
-    # TODO support caching?
-    #request_caching_args = request_caching_arg_to_dict(
-    #    cache_requests=args.cache_requests
-    #)
+    if ',' in args.filter:
+        filter_list = args.filter.split(',')
+        if len(filter_list) != len(task_names):
+            raise ValueError(
+                f"Filter list length {len(filter_list)} does not match task list length {len(task_list)}. Provide one filter per task."
+            )
+    else:
+        filter_list = [args.filter]
+
+    # set up scheduler
+    if args.file_schedule is not None and args.sampling_schedule is not None:
+        logging.error("Cannot set file_schedule and sampling_schedule at same time.")
+        sys.exit(1)
+
+    if args.file_schedule is not None:
+        logging.info("Using {args.file_schedule} for match schedule.")
+        scheduler = FileScheduler(args.file_schedule)
+    elif args.sampling_schedule is not None and args.sampling_schedule:
+
+        if args.match_size is None:
```

Comment:
both `match_size` and num_rounds` default to `1` so these error conditionals are not needed

Reply:
Yup. Removing those conditionals now.